src/app/layout/qgslayoutdesignerdialog.h

```diff
@@ -563,8 +563,8 @@ class QgsLayoutDesignerDialog: public QMainWindow, public Ui::QgsLayoutDesignerB
     void showRasterizationWarning();
     void showForceVectorWarning();
 
-    bool showFileSizeWarning();
-    bool getRasterExportSettings( QgsLayoutExporter::ImageExportSettings &settings, QSize &imageSize );
+    bool showFileSizeWarning()   ;
+    bool getRasterExportSettings( QgsLayoutExporter::ImageExportSettings &settings, QSize &imageSize, const QString &fileExtension = QString() );
```

Comment:
We can safely modify the signature here, as this method isn't exposed to python:

```suggestion
    bool getRasterExportSettings( QgsLayoutExporter::ImageExportSettings &settings, QSize &imageSize, const QString &fileExtension );
```